backtester/funding/item.go

```diff
@@ -92,11 +92,11 @@ func (i *Item) Equal(item *Item) bool {
 }
 
 // BasicEqual checks for equality via passed in values
-func (i *Item) BasicEqual(exch string, a asset.Item, currency, pairedCurrency currency.Code) bool {
+func (i *Item) BasicEqual(exch string, a asset.Item, ccy, pairedCurrency currency.Code) bool {
```

Comment:
🐒 I think `curr` is much more widely accepted. It has the advantage of being prefixable and uppercasable, case-in-point with `pairedCurr` here too.
```suggestion
func (i *Item) BasicEqual(exch string, a asset.Item, curr, pairedCurr currency.Code) bool {
```